src/main/java/org/team1540/robot2025/subsystems/intake/IntakeIOReal.java

```diff
@@ -0,0 +1,76 @@
+package org.team1540.robot2025.subsystems.intake;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.PositionVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.spark.SparkLowLevel;
+import com.revrobotics.spark.SparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.team1540.robot2025.Constants;
+
+public class IntakeIOReal implements IntakeIO {
+
+    // rotation of horizontal beams for the intake, clockwise to intake, counter-clockwise to spit out
+    private final TalonFX topFalcon = new TalonFX(Constants.IntakeConstants.TOP_FALCON_ID);
+
+    // controls height of intake
+    private final TalonFX bottomFalcon = new TalonFX(Constants.IntakeConstants.BOTTOM_FALCON_ID);
```

Comment:
Rather than top and bottom, labeling these (and related fields) with what they do would be clearer.